modules/components/pages/processors/aws_bedrock_chat.adoc

```diff
@@ -0,0 +1,236 @@
+= aws_bedrock_chat
+:type: processor
+:status: experimental
+:categories: ["AI"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Generates responses to messages in a chat conversation, using the AWS Bedrock API.
```

Comment:
consider linking to this: https://aws.amazon.com/bedrock/

Reply:
updated